Makefile

```diff
@@ -8,6 +8,7 @@
 .PHONY: benchmark-store, drone, check-mod
 
 SHELL = /usr/bin/env bash
+MOD_VENDOR=-mod=vendor
```

Comment:
I think you could make this `?=` and then instead of having to modify the make file you could run 

`make MOD_VENDOR= all`

Reply:
And then I'm wondering if a conditional would be better:

```suggestion
VENDOR?=true
MOD_VENDOR=
ifeq ($(VENDOR),true)
  MOD_VENDOR=-mod=vendor
endif
```

I'm not sure that syntax is all correct, I have to google makefile stuff whenever I work with them

Reply:
Yes, that would be nicer, although just running `MOD_VENDOR=  make all` works fine as well. But I'll try to `VENDOR=false` work.

Reply:
So I basically took your code, tested it (it works! :)) and committed.

Reply:
`VENDOR=false make all` disables vendoring, while `VENDOR=true make all` or just `make all` enables it.